### ISSUE TITLE
bad linting of switch statement

### ISSUE THREAD

**Issue by fiisch** [s-shin/linter-perl#38]
Consider this code (minimal reproducible case):
```perl
#!/usr/bin/perl
use strict;
use warnings;
use Switch;
package Something;
my $a = 0;
switch($a) {          #this line is marked as error
  case 0 { print "0\n" }
  case 1 { print "1\n" }
  else { print "2\n" }
}
1;
```
The linter gives error:
```
Error	B::Lint	Illegal reference to private name '_S_W_I_T_C_H'	7:1
```
However running B:Lint explicitly, it says everything is ok (also, code runs fine):
```
fiisch@seeker:~/git-repo/perl-dt2g (master %)> perl -MO=Lint minimal_case.pl
minimal_case.pl syntax OK
```
When I comment out the "package Something;" and "1;" - making it ordinary perl script, the linter problem goes away.
This behavior is quite annoying because I cannot simply look at the code and "see no red, syntax is therefore ok". Could you please look into it?

I am using atom 1.14.4 x64 with linter-perl 0.8.1 and linter 2.2.0 packages. OS is openSuSE 42.1.

**Comment by s-shin** [comment]
Could you get the error by executing `perl -MO=Lint,all minimal_case.pl`.

Since the value of `linter-perl.lintOptions` is `all` by default, the linter will check all rules that include the `private_name` rule.

**Comment by fiisch** [comment]
Yes, you are right, the `perl -MO=Lint,all minimal_case.pl` reports the switch statement.
As a try, I set `lintOptions: ""` in the Atom config and all looks good now. Is this the correct way to do it, or does setting the empty string break something in the mod?
